src/components/MobileHeader/MobileHeader.jsx

```diff
@@ -142,7 +143,7 @@ const MobileHeader = ({
           </div>
         )}
       </div>
-      <div className={mobileHeaderMenuClassName} ref={menu}>
+      <div className={mobileHeaderMenuClassName} data-ds-override='focus' ref={menu}>
```

Comment:
Could we add a custom class-name and target that here instead of a custom attribute?  Our downstream sites don't get along with the custom attributes.

Reply:
requested changes applied :)